web/templates/web/about.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'web/base.html' %}
```

Comment:
Why did you add the `web/` back into these and move the files into a `web` directory?

Reply:
When I was running some test to make sure that  the view was actually using the correct template, I ran into some errors. The test didn't match any if I just wrote the template name. therefore, I google it, and find out the the best approach was to add the namespacing, and then add the template path to the assertTemplateUsed(some_response, template path).